src/pineko/cli/convolute.py

```diff
@@ -1,6 +1,8 @@
 # -*- coding: utf-8 -*-
 """CLI entry point to convolution."""
 import click
+import pineappl as pine
```

Comment:
let's not change the `pineappl` import but rather rename the conflicting argument